Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -738,7 +738,7 @@ public extension ChatChannelController {
             return
         }
 
-        guard let messageId = messageId ?? lastFetchedMessageId else {
```

Comment:
Keep in mind that with this, if the `messages.last?.id` message has failed sending (Is not in the backend), this issue will still be valid: https://github.com/GetStream/stream-chat-swift/issues/2229

So what we actually need is something like:
`messages.last { where message not failed }?.id`

Reply:
Good catch, will take it into consideration